rued-esp32/src/peripherals.rs

```diff
@@ -66,17 +70,35 @@ impl
     pub fn take() -> Self {
         let peripherals = Peripherals::take().unwrap();
 
+        let spi1 = SpiPeripherals {
+            spi: peripherals.spi2,
+            sclk: peripherals.pins.gpio18.into(), // SCK
+            sdo: peripherals.pins.gpio23.into(),  // COPI
+            // NOTE: Display does not need a valid SDI pin; therefore
+            // settings this to an unused GPIO.
+            sdi: peripherals.pins.gpio10.into(), // dummy (SDI is not needed)
+            cs: Some(peripherals.pins.gpio5.into()), // CS
+        };
```

Comment:
This is for the display, as I've got my testing board wired up in this manner - and these pins are valid for the MicroMod ESP32 processor board.